pages/common/clang.md

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
I think the paths must also be updated here to `path/to/input_source.c`.

Reply:
Yes, they must. It's a problem to fix such problems automatically as  `source` is not a standardized word for file placeholders.

Reply:
Fixed automatically.